apps/nova/src/constants/ui.ts

```diff
@@ -0,0 +1 @@
+export const HEADER_HEIGHT = 106;
```

Comment:
First of all, it is super nice that you made use of the `constants/` folder. 

My question is what do you mean by `HEADER`? Do you mean the navbar? Also in which unit it is? 

To make it clear which unit it is we can do one of the following: 

```ts
export const HEADER_HEIGHT = '106px'; 
export const HEADER_HEIGHT_IN_PIXELS = 106;
/**
 * Header height in pixels.
 */
export const HEADER_HEIGHT = 106;
```